integrationExamples/gpt/x-domain/creative.js

```diff
@@ -0,0 +1,53 @@
+// this script can be returned by an ad server delivering a cross domain iframe, into which the
+// creative will be rendered, e.g. DFP delivering a SafeFrame
```

Comment:
Can make a note about using a macro here instead of a hardcoded string. Many creatives will be delivered to multiple domains.